.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "libssh2-sys/libssh2"]
 	path = libssh2-sys/libssh2
-	url = https://github.com/wez/libssh2
-	branch = pr484
+	url = https://github.com/libssh2/libssh2
+	branch = master
```

Comment:
I think this is right, but submodules can be tricky.  I'd expect to see the submodule changes as part of this PR (eg: take a look at an earlier commit: https://github.com/alexcrichton/ssh2-rs/commit/e8d4dc165fbcd3fceeec96db9d1e9be9b67bef7b)

It's worth re-syncing the git modules and making sure that you've checked out the appropriate commit from master in `libssh2-sys/libssh2`, then `git add libssh2-sys/libssh2` and include it in this PR.

Reply:
I hate git submodules to be honest :cry:  I think I missed adding the submodule dir and committing also that. Let me see if I can make something closer to the other PR. Thanks for the feedback.

Reply:
I think I did it..